src/assets/styles/components/_nav.scss

```diff
@@ -114,15 +114,23 @@
 .menu a,
 .menu button {
   align-items: center;
+  border: 2px solid;
+  border-radius: 5px;
   display: flex;
   font-family: var(--fl-font-family, var(--ff-display));
   font-weight: var(--fw-semibold);
   height: rem(50);
+  outline: none;
```

Comment:
See above!

Reply:
Also, this breaks the hover styles for the main navigation bar:

<img width="377" alt="Screenshot 2023-06-27 at 10 47 43 AM" src="https://github.com/inclusive-design/idrc/assets/605361/0864b8d5-d46b-4a0b-9a40-70b17cb0e68b">